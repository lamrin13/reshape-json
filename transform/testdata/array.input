[
    {
        "name": "John",
        "email": "john.a@abc.com"
    },
    {
        "name": "Bob",
        "email": "bob.b@abc.com"
    }
]