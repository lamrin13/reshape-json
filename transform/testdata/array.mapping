{
    "user.Name": "name",
    "user.emailAddress": "email"
}