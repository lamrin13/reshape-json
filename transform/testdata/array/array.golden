[
    {
        "user": {
            "Name": "John",
            "emailAddress": "john.a@abc.com"
        }
    },
    {
        "user": {
            "Name": "Bob",
            "emailAddress": "bob.b@abc.com"
        }
    }
]